src/DynamoCoreWpf/UI/SharedResourceDictionary.cs

```diff
@@ -64,6 +64,8 @@ public static class SharedDictionaryManager
         private static ResourceDictionary _connectorsDictionary;
         private static ResourceDictionary _portsDictionary;
         private static ResourceDictionary _sidebarGridDictionary;
+        private static ResourceDictionary _outPortsDictionary;
```

Comment:
Can we not have the underscore anymore? I don't think it's in our coding convention.

Reply:
I was still getting warning from visual studio if I rename it